src/notation/qml/MuseScore/NotationScene/internal/ElementPopupLoader.qml

```diff
@@ -71,7 +72,13 @@ Item {
     function show(elementType, elementRect) {
         close()
 
-        var popup = loader.loadPopup(prv.componentByType(elementType), elementRect)
+        var component = prv.componentByType(elementType)
+
+        var popup = loader.loadPopup(component, elementRect)
+        if (!popup.model.canOpen) {
```

Comment:
It would be more ideal to make this decision earlier, in C++, to avoid needlessly creating and destroying popup windows.